<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSourceConfigurationEditor</class>
 <widget class="QWidget" name="DataSourceConfigurationEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Decreasing color:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Increasing color: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Pen color:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="tbDecreasingColor"/>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="tbIncreasingColor"/>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="tbPenColor"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
